README.md

```diff
@@ -15,7 +15,7 @@ package { 'firefox': provider => 'brewcask' }
 
 ## Required Puppet Modules
 
- - `homebrew`, >= 1.10.0
+ - `homebrew` >= 1.10.0
 
 ## Work in progress
```

Comment:
I know this wasn't you but can you delete this section? I don't think it really adds any value or information.

Reply:
OK, I same as [puppuet-homebrew/README](https://raw.githubusercontent.com/boxen/puppet-homebrew/master/README.md)